index.bs

```diff
@@ -293,6 +293,14 @@ Parts of the web platform evolve independently.
 Issues that are present with a certain web technology now may be fixed in a subsequent iteration.
 Duplicating these issues makes fixing them more difficult.
 By adhering to this principle we can make sure overall platform quality improves over time.
+
+<h3 id="data-minimization">Minimize data</h3>
```

Comment:
We probably want to indicate better our intent of **what** data - the intent here is likely user data.

Reply:
We discussed this in the breakout earlier and concluded this is generally applicable to any data in this context, rather than specifically in user data

Reply:
Subsequent discussion that all data relevant in this context is user data